packages/postgres-database/src/simcore_postgres_database/models/resource_tracker_pricing_details.py

```diff
@@ -38,11 +38,10 @@
         sa.String,
         nullable=False,
         doc="The custom name of the pricing plan, ex. DYNAMIC_SERVICES_TIERS, COMPUTATIONAL_SERVICES_TIERS, CPU_HOURS, STORAGE",
-        index=True,
     ),
     sa.Column(
         "cost_per_unit",
-        sa.Numeric(precision=3, scale=2),
+        sa.Numeric(precision=15, scale=2),
```

Comment:
hahah! I have just noticed that myself while looking at your code. I was about to test  whether we can remove `precision` ...

Reply:
let's add in _common a common numeric definition to use the same eerywhere?